utils.py

```diff
@@ -93,6 +129,38 @@ def get_full_kernel_path(kernel_location, image, arch=None):
     return kernel.resolve()
 
 
+def get_gh_json(endpoint):
+    """
+    Query a GitHub API endpoint.
+
+    Parameters:
+        endpoint (str): The URL of the endpoint to query.
+
+    Returns:
+        A JSON object from the result of the query.
+    """
+    curl_cmd = ['curl', '-LSs']
+    if 'GITHUB_TOKEN' in os.environ:
+        curl_cmd += [
+            '-H',
+            'Accept: application/vnd.github+json',
+            '-H',
+            f"Authorization: Bearer {os.environ['GITHUB_TOKEN']}",
```

Comment:
Are there docs you can add a comment linking to for these parameters?

Reply:
Done: 3c6a83f7e876a06f819d40658939d07e7a799929